src/open_company_web/components/topic_list.cljs

```diff
@@ -179,7 +177,133 @@
      :fullscreen-force-edit (if (nil? current-state) (router/section-editing?) (:fullscreen-force-edit current-state))
      :show-add-topic-tip show-add-topic-tip
      :show-second-add-topic-tooltip (or (:show-second-add-topic-tooltip current-state) false)
-     :show-share-tooltip (or (:show-share-tooltip current-state) false)}))
+     :show-share-tooltip (or (:show-share-tooltip current-state) false)
+     :show-second-pin-tip (or (:show-second-pin-tip current-state) false)
+     :rerender (rand 4)}))
+
+(defn save-sections-order [owner]
```

Comment:
function docstring and possibly some inline comments would be good here. this function is non-trivial.